examples/terraform-submodules/gke/module.tf

```diff
@@ -40,8 +40,8 @@ variable "node_count" {
 }
 
 module "agones" {
-  source = "git::https://github.com/googleforgames/agones.git//build/?ref=master"
-  
+  source = "git::https://github.com/googleforgames/agones.git//install/terraform/?ref=master"
```

Comment:
Change to `?ref=release-1.2.0`.

Reply:
Or we can leave this as master until we cut the 1.2.0 release candidate (so it works in the mean time). 

@markmandel - wdyt?

Reply:
Just so I understand - assuming it points to 1.2.0, does that mean this file would need to be updated on every release? (in which case, there should be an item in the release checklist somewhere?)

Reply:
Yes, it would need to be updated each release (and the PR includes a change to the release checklist).

Reply:
Since there is an item in the release checklist, I'm okay with it being master until we hit RC. No objections here. :+1:

Reply:
Leaving the all references to the master branch for consistency, unless objections.